src/components/pages/MatchUp/FaceoffContent.js

```diff
@@ -127,8 +127,10 @@ const FaceoffSubDisplay = ({ sub, type, feedback, ...props }) => {
         <div className="submission-preview">
           {!locked ? (
             <img
+              className="cursor-pointer"
               src={type === 'DRAWING' ? sub.ImgURL : sub.Pages[0].PageURL}
               alt="text"
+              role="button"
```

Comment:
I went ahead and added role="button" for a11y purposes. Any img tags you see that have an onClick should have this role.

Reply:
Good call, I didn't think of that